modules/qna/src/views/FormModal.jsx

```diff
@@ -83,7 +83,7 @@ export default class FormModal extends Component {
     const { item, isText, isRedirect } = this.state
     const invalidFields = {
       questions: !item.questions.length || !item.questions[0].length,
-      answer: isText && (!item.answers.length || !item.answers[0].length),
+      answer: isText && (!item.answers.length || !(typeof item.answers[0].length == 'string' ? item.answers[0].length : true)),
```

Comment:
This condition isn't very clear (typeof .length === string?) Also missing strict check. Can you extract it in a meaningful text?